modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -92,7 +116,12 @@ installation, do not delete these files.
 . Configure the network boot infrastructure so that the machines boot from their
 local disks after {op-system} is installed on them.
 
-. Configure PXE or iPXE installation for the {op-system} images and begin the installation.
+ifndef::only-pxe[]
+. Configure PXE or iPXE installation for the {op-system} images.
+endif::only-pxe[]
+ifdef::only-pxe[]
+. Configure PXE installation for the {op-system} images.
```

Comment:
You could add "and begin the installation" at the end of this line, to clarify that the installation should start here. This provides consistency with the subsequent step.